crma_api_client/resources/dataset.py

```diff
@@ -0,0 +1,42 @@
+"""Contains definitions for the dataset resource"""
+
+from datetime import datetime
+from typing import List
+
+from pydantic import BaseModel
+
+from .user import User
+from .util import to_camel
+
+
+class Dataset(BaseModel):
+    """Dataset model"""
+
+    id: str
+    url: str
+
+
+class DatasetVersion(BaseModel):
+    """Dataset version model"""
```

Comment:
since this a public repo I wonder if there is value in linking to the API docs for this model?